Source/DafnyStandardLibraries/Makefile

```diff
@@ -10,6 +10,10 @@ DOO_FILE_ARITHMETIC_TARGET=binaries/DafnyStandardLibraries-arithmetic.doo
 
 all: check-binary test check-format check-examples
 
+verify:
```

Comment:
Nice improvements! I had some of these as part of the FileIO import too, especially since the build will change per language too. I did find a slightly cleaner approach but I can apply that in my PR instead of insisting on it here.